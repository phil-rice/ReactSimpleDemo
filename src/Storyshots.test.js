import initStoryshots from '@storybook/addon-storyshots';

if (process.env.NOSNAPSHOTS) {
    describe("behavior of snapshots when disabled", () => {
        it("they don't do anything", () => {
        })
    })
} else {
    initStoryshots();
}