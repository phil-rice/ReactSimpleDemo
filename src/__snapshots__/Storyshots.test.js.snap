// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/Buttons/Button Primary 1`] = `
<button
  id="1"
>
  Title One
</button>
`;

exports[`Storyshots Components/Buttons/Toggle Primary 1`] = `
<button
  onClick={[Function]}
>
  title1
</button>
`;

exports[`Storyshots Data/AttributeValues/TitleAndValues With Title 1`] = `
<div>
  <div>
    <h2
      className="title"
    >
      Some Title
    </h2>
    <dl
      className="table-display"
    >
      <dt>
        Title One
      </dt>
      <dd>
        Value One
      </dd>
      <dt>
        Title Two
      </dt>
      <dd>
        Value Two
      </dd>
    </dl>
  </div>
</div>
`;

exports[`Storyshots Data/AttributeValues/TitleAndValues Without Title 1`] = `
<div>
  <div>
    <dl
      className="table-display"
    >
      <dt>
        Title One
      </dt>
      <dd>
        Value One
      </dd>
      <dt>
        Title Two
      </dt>
      <dd>
        Value Two
      </dd>
    </dl>
  </div>
</div>
`;

exports[`Storyshots Data/AttributeValues/TitleValueComp With Title 1`] = `
<div>
  <div
    className=""
  >
    <h2
      className="title"
    >
      Some Title
    </h2>
    <dl>
      <dd>
        name1
      </dd>
      <dt>
        value1
      </dt>
      <dd>
        name2
      </dd>
      <dt>
        value2
      </dt>
      <dd>
        name3
      </dd>
      <dt>
        value3
      </dt>
    </dl>
  </div>
</div>
`;

exports[`Storyshots Data/AttributeValues/TitleValueComp Without Title 1`] = `
<div>
  <div
    className=""
  >
    <dl>
      <dd>
        name1
      </dd>
      <dt>
        value1
      </dt>
      <dd>
        name2
      </dd>
      <dt>
        value2
      </dt>
      <dd>
        name3
      </dd>
      <dt>
        value3
      </dt>
    </dl>
  </div>
</div>
`;

exports[`Storyshots Data/AttributeValues/Values With Title 1`] = `
<div>
  <h2
    className="title"
  >
    Some Title
  </h2>
  <div
    className="value"
  >
    Value One
  </div>
  <div
    className="value"
  >
    Value Two
  </div>
</div>
`;

exports[`Storyshots Data/AttributeValues/Values Without Title 1`] = `
<div>
  <div
    className="value"
  >
    Value One
  </div>
  <div
    className="value"
  >
    Value Two
  </div>
</div>
`;

exports[`Storyshots Data/Buttons/ButtonTitleValue Primary 1`] = `
<button
  id="1"
>
  Title One
  :
  Value One
</button>
`;

exports[`Storyshots Data/Titles/OptionalTitle With Out Title 1`] = `null`;

exports[`Storyshots Data/Titles/OptionalTitle With Title 1`] = `
<h2
  className="title"
>
  Some Title
</h2>
`;

exports[`Storyshots Data/Titles/WrapTitle With Out Title 1`] = `
<div>
  <p>
    Here is the wrapped Component
  </p>
</div>
`;

exports[`Storyshots Data/Titles/WrapTitle With Title 1`] = `
<div>
  <h2
    className="title"
  >
    Some Title
  </h2>
  <p>
    Here is the wrapped Component
  </p>
</div>
`;

exports[`Storyshots Examples/AccountPersonalisation Loading False 1`] = `
<div
  className="page"
>
  <div>
    <h1>
      Account Personalisation
    </h1>
  </div>
  <div>
    <div>
      <div
        className="columns"
      >
        <div
          className=""
        >
          <dl>
            <dd>
              status
            </dd>
            <dt>
              Received Complete
            </dt>
            <dd>
              estimatedCompletionDate
            </dd>
            <dt>
              2020-10-01
            </dt>
          </dl>
        </div>
        <div
          className=""
        >
          <dl>
            <dd>
              sortCode
            </dd>
            <dt>
              10-11-12
            </dt>
            <dd>
              bankAccountNumber
            </dd>
            <dt>
              12334353
            </dt>
            <dd>
              lastChanged
            </dd>
            <dt>
              2020-09-01
            </dt>
          </dl>
        </div>
      </div>
    </div>
    <div>
      <div
        className="columns"
      >
        <div
          className=""
        >
          <h2
            className="title"
          >
            Books/cards
          </h2>
          <dl>
            <dd>
              personalisation
            </dd>
            <dt>
              Mr Fred Bloggs
            </dt>
            <dd>
              title
            </dd>
            <dt>
              Mr
            </dt>
            <dd>
              forename
            </dd>
            <dt>
              Fred
            </dt>
            <dd>
              middleName
            </dd>
            <dt>
              
            </dt>
            <dd>
              surname
            </dd>
            <dt>
              Bloggs
            </dt>
            <dd>
              personalisation
            </dd>
            <dt>
              Mr Fred Bloggs
            </dt>
          </dl>
        </div>
        <div
          className=""
        >
          <dl>
            <dd>
              personalisation
            </dd>
            <dt>
              Mrs Matilda Bloggs
            </dt>
            <dd>
              title
            </dd>
            <dt>
              Mrs
            </dt>
            <dd>
              forename
            </dd>
            <dt>
              Matilda
            </dt>
            <dd>
              middleName
            </dd>
            <dt>
              
            </dt>
            <dd>
              surname
            </dd>
            <dt>
              Bloggs
            </dt>
            <dd>
              personalisation
            </dd>
            <dt>
              Mrs Matilda Bloggs
            </dt>
          </dl>
        </div>
      </div>
    </div>
    <div>
      <div
        className="columns"
      >
        <p>
          Statements
        </p>
        <p>
          First on fullfilment
        </p>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Examples/AccountPersonalisation Loading True 1`] = `
<div
  className="page"
>
  <div>
    <h1>
      Account Personalisation
    </h1>
  </div>
  <p>
    Loading
  </p>
</div>
`;

exports[`Storyshots Examples/Statement/StatementPage Loading False 1`] = `
<div
  className="page"
>
  <div>
    <h1>
      MainStatement
    </h1>
  </div>
  <div>
    <div>
      <h2
        className="title"
      >
        Regular Statement
      </h2>
      <div
        className="columns"
      >
        <div>
          <h2
            className="title"
          >
            Statement Address
          </h2>
          <div
            className="value"
          >
            4 Privet Drive
          </div>
          <div
            className="value"
          >
            Little Whinging
          </div>
          <div
            className="value"
          >
            
          </div>
          <div
            className="value"
          >
            Surrey
          </div>
          <div
            className="value"
          >
            LW10 3SD
          </div>
        </div>
        <div>
          <dl
            className="table-display"
          >
            <dt>
              Statement Frequency
            </dt>
            <dd>
              Monthly
            </dd>
            <dt>
              Last Statement Date
            </dt>
            <dd>
              31st March 2021
            </dd>
            <dt>
              Last Statement No
            </dt>
            <dd>
              00031
            </dd>
            <dt>
              Next Statement Date
            </dt>
            <dd>
              30th April 2021
            </dd>
          </dl>
        </div>
      </div>
    </div>
    <div>
      <h2
        className="title"
      >
        Interim Statement
      </h2>
      <div
        className="columns"
      >
        <button
          id="buttonRequestInterim"
        >
          Request Interim Payment
        </button>
        <button
          id="buttonNextStatement"
        >
          Last Statement Date
          :
          31st March 2021
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Examples/Statement/StatementPage Loading True 1`] = `
<div
  className="page"
>
  <div>
    <h1>
      MainStatement
    </h1>
  </div>
  <p>
    Loading
  </p>
</div>
`;

exports[`Storyshots Examples/Statement/StatementPage2x2 Loading False 1`] = `
<div
  className="page"
>
  <div>
    <h1>
      2x2Statement
    </h1>
  </div>
  <div>
    <div>
      <h2
        className="title"
      >
        Regular Statement
      </h2>
      <div
        className="columns"
      >
        <div>
          <h2
            className="title"
          >
            Statement Address
          </h2>
          <div
            className="value"
          >
            4 Privet Drive
          </div>
          <div
            className="value"
          >
            Little Whinging
          </div>
          <div
            className="value"
          >
            
          </div>
          <div
            className="value"
          >
            Surrey
          </div>
          <div
            className="value"
          >
            LW10 3SD
          </div>
        </div>
        <div>
          <dl
            className="table-display"
          >
            <dt>
              Statement Frequency
            </dt>
            <dd>
              Monthly
            </dd>
            <dt>
              Last Statement Date
            </dt>
            <dd>
              31st March 2021
            </dd>
            <dt>
              Last Statement No
            </dt>
            <dd>
              00031
            </dd>
            <dt>
              Next Statement Date
            </dt>
            <dd>
              30th April 2021
            </dd>
          </dl>
        </div>
      </div>
    </div>
    <div>
      <h2
        className="title"
      >
        Interim Statement
      </h2>
      <div
        className="columns"
      >
        <button
          id="buttonRequestInterim"
        >
          Request Interim Payment
        </button>
        <button
          id="buttonNextStatement"
        >
          Last Statement Date
          :
          31st March 2021
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Examples/Statement/StatementPage2x2 Loading True 1`] = `
<div
  className="page"
>
  <div>
    <h1>
      2x2Statement
    </h1>
  </div>
  <p>
    Loading
  </p>
</div>
`;

exports[`Storyshots Layout/Columns/TwoColumnPanel With Title 1`] = `
<div>
  <h2
    className="title"
  >
    Some Title
  </h2>
  <div
    className="columns"
  >
    <p>
      Child1
    </p>
    <p>
      Child2
    </p>
  </div>
</div>
`;

exports[`Storyshots Layout/Columns/TwoColumnPanel Without Title 1`] = `
<div>
  <div
    className="columns"
  >
    <p>
      Child1
    </p>
    <p>
      Child2
    </p>
  </div>
</div>
`;

exports[`Storyshots Layout/Grid/TwoByThree With Title 1 And 2 And 3 1`] = `
<div>
  <div>
    <h2
      className="title"
    >
      Some Title 1
    </h2>
    <div
      className="columns"
    >
      <p>
        Child1
      </p>
      <p>
        Child2
      </p>
    </div>
  </div>
  <div>
    <h2
      className="title"
    >
      Some Title 2
    </h2>
    <div
      className="columns"
    >
      <p>
        Child3
      </p>
      <p>
        Child4
      </p>
    </div>
  </div>
  <div>
    <h2
      className="title"
    >
      Some Title 3
    </h2>
    <div
      className="columns"
    >
      <p>
        Child5
      </p>
      <p>
        Child6
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Layout/Grid/TwoByTwo With Title 1 1`] = `
<div>
  <div>
    <h2
      className="title"
    >
      Some Title 1
    </h2>
    <div
      className="columns"
    >
      <p>
        Child1
      </p>
      <p>
        Child2
      </p>
    </div>
  </div>
  <div>
    <div
      className="columns"
    >
      <p>
        Child3
      </p>
      <p>
        Child4
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Layout/Grid/TwoByTwo With Title 1 And 2 1`] = `
<div>
  <div>
    <h2
      className="title"
    >
      Some Title 1
    </h2>
    <div
      className="columns"
    >
      <p>
        Child1
      </p>
      <p>
        Child2
      </p>
    </div>
  </div>
  <div>
    <h2
      className="title"
    >
      Some Title 2
    </h2>
    <div
      className="columns"
    >
      <p>
        Child3
      </p>
      <p>
        Child4
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Layout/Grid/TwoByTwo With Title 2 1`] = `
<div>
  <div>
    <div
      className="columns"
    >
      <p>
        Child1
      </p>
      <p>
        Child2
      </p>
    </div>
  </div>
  <div>
    <h2
      className="title"
    >
      Some Title 2
    </h2>
    <div
      className="columns"
    >
      <p>
        Child3
      </p>
      <p>
        Child4
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Layout/Grid/TwoByTwo Without Title 1`] = `
<div>
  <div>
    <div
      className="columns"
    >
      <p>
        Child1
      </p>
      <p>
        Child2
      </p>
    </div>
  </div>
  <div>
    <div
      className="columns"
    >
      <p>
        Child3
      </p>
      <p>
        Child4
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Layout/Rows/ThreeRowPanel With Title 1`] = `
<div>
  <h2
    className="title"
  >
    Some Title
  </h2>
  <p>
    Child1
  </p>
  <p>
    Child2
  </p>
  <p>
    Child3
  </p>
</div>
`;

exports[`Storyshots Layout/Rows/ThreeRowPanel Without Title 1`] = `
<div>
  <p>
    Child1
  </p>
  <p>
    Child2
  </p>
  <p>
    Child3
  </p>
</div>
`;

exports[`Storyshots Layout/Rows/TwoRowPanel With Title 1`] = `
<div>
  <h2
    className="title"
  >
    Some Title
  </h2>
  <p>
    Child1
  </p>
  <p>
    Child2
  </p>
</div>
`;

exports[`Storyshots Layout/Rows/TwoRowPanel Without Title 1`] = `
<div>
  <p>
    Child1
  </p>
  <p>
    Child2
  </p>
</div>
`;

exports[`Storyshots Loading/Loading Loading False 1`] = `
<p>
  The Component is now being displayed
</p>
`;

exports[`Storyshots Loading/Loading Loading True 1`] = `
<p>
  Loading
</p>
`;

exports[`Storyshots Nav/SelectPage Select Statement 1`] = `
<button
  onClick={[Function]}
>
  Statement
</button>
`;

exports[`Storyshots Page/Loading With Out Title Loading 1`] = `
<div
  className="page"
>
  <div>
    <h1 />
  </div>
  <p>
    Loading
  </p>
</div>
`;

exports[`Storyshots Page/Loading With Out Title Not Loading 1`] = `
<div
  className="page"
>
  <div>
    <h1 />
  </div>
  <p>
    The page goes here
  </p>
</div>
`;

exports[`Storyshots Page/Loading With Title Loading 1`] = `
<div
  className="page"
>
  <div>
    <h1>
      The Page Title
    </h1>
  </div>
  <p>
    Loading
  </p>
</div>
`;

exports[`Storyshots Page/Loading With Title Not Loading 1`] = `
<div
  className="page"
>
  <div>
    <h1>
      The Page Title
    </h1>
  </div>
  <p>
    The page goes here
  </p>
</div>
`;

exports[`Storyshots Page/Page With Out Title 1`] = `
<div
  className="page"
>
  <div>
    <h1 />
  </div>
  <p>
    The page goes here
  </p>
</div>
`;

exports[`Storyshots Page/Page With Title 1`] = `
<div
  className="page"
>
  <div>
    <h1>
      The Page Title
    </h1>
  </div>
  <p>
    The page goes here
  </p>
</div>
`;
